internal/services/network/express_route_circuit_resource.go

```diff
@@ -140,6 +140,12 @@ func resourceExpressRouteCircuit() *pluginsdk.Resource {
 				ValidateFunc:  expressrouteports.ValidateExpressRoutePortID,
 			},
 
+			"enable_rate_limiting": {
```

Comment:
for consistency within the provider, could we rename this `rate_limiting_enabled`?